tsconfig.json

```diff
@@ -5,5 +5,5 @@
     "outDir": "./build"
   },
   "include": ["./src"],
-  "exclude": ["**/__mocks__/*", "**/__tests__/*"]
+  "exclude": ["**/__mocks__/*", "**/__tests__/*", "**/__rsc_tests__/*"]
```

Comment:
Not sure if that change is necessary.

Reply:
It was auto-added after upgrading the packages. I assumed it's a part of boilerplate, but considering some of the Expo packages don't include this I'll remove here.

Reply:
This file was regenerated by `expo-module-scripts` after the upgrade and matches corresponding `tsconfig.json`'s of `expo-image-picker` and `expo-media-library`. I don't know the exact purpose of `/__rsc_tests__/`, but I trust the judgement of `expo-module-scripts` with regards to its exclusion.